code/datums/wounds/pierce.dm

```diff
@@ -23,36 +30,51 @@
 	/// If we let off blood when hit, the max blood lost is this * the incoming damage
 	var/internal_bleeding_coefficient
 
+//ok so this is ultra stupid, but we need to update examine and shit in case the owner is not a bleeder
+/datum/wound/pierce/apply_wound(obj/item/bodypart/L, silent, datum/wound/old_wound, smited, attack_direction)
+	//also it's ok to not typecheck, humans are the only ones that deal with wounds
+	var/mob/living/carbon/human/human_victim = L?.owner
+	if(human_victim)
+		no_bleeding = !(NOBLOOD in human_victim.dna.species.species_traits)
```

Comment:
Not having NOBLOOD means that we do have blood right?

Reply:
good catch, how did i miss that